ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -138,66 +125,13 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
-                    <Route path={compliancePath} component={AsyncCompliancePage} />
-                    <Route
-                        path={complianceEnhancedBasePath}
-                        component={AsyncComplianceEnhancedPage}
-                    />
-                    <Route path={integrationsPath} component={AsyncIntegrationsPage} />
-                    <Route path={policyManagementBasePath} component={AsyncPolicyManagementPage} />
                     {/* Make sure the following Redirect element works after react-router-dom upgrade */}
                     <Redirect exact from={deprecatedPoliciesPath} to={policiesPath} />
-                    {hasCollectionsPermission && (
-                        <Route path={collectionsPath} component={AsyncCollectionsPage} />
-                    )}
-                    <Route path={riskPath} component={AsyncRiskPage} />
-                    <Route path={accessControlPath} component={AsyncAccessControlPageV2} />
-                    <Route path={searchPath} component={AsyncSearchPage} />
-                    <Route path={apidocsPath} component={AsyncApiDocsPage} />
-                    <Route path={userBasePath} component={AsyncUserPage} />
-                    <Route path={systemConfigPath} component={AsyncSystemConfigPage} />
-                    {isVulnMgmtWorkloadCvesEnabled && (
-                        <Route
-                            path={vulnerabilitiesWorkloadCvesPath}
-                            component={AsyncWorkloadCvesPage}
-                        />
-                    )}
-                    {hasVulnerabilityReportsPermission &&
-                        isVulnerabilityReportingEnhancementsEnabled && (
-                            <Route
-                                path={vulnerabilityReportsPath}
-                                component={AsyncVulnerabilityReportingPage}
-                            />
-                        )}
-                    {hasVulnerabilityReportsPermission && (
-                        <Route path={vulnManagementReportsPath} component={AsyncVulnMgmtReports} />
-                    )}
-                    <Route
-                        path={vulnManagementRiskAcceptancePath}
-                        component={AsyncVulnMgmtRiskAcceptancePage}
-                    />
-                    <Route path={vulnManagementPath} component={AsyncVulnMgmtPage} />
-                    <Route path={configManagementPath} component={AsyncConfigManagementPage} />
-                    <Route
-                        path={clustersDelegateScanningPath}
-                        component={AsyncDelegateScanningPage}
-                    />
-                    <Route path={clustersPathWithParam} component={AsyncClustersPage} />
-                    {process.env.NODE_ENV === 'development' && (
-                        <Route path={clustersListPath} component={AsyncPFClustersPage} />
-                    )}
```

Comment:
Although I think removing this route and the dependency on `process.env.NODE_ENV` is a good idea, it makes me wonder what possible extensions we could add to the `isRouteEnabled` abstraction to handle odd cases like this in the future.